runners/metrics_client_test.go

```diff
@@ -35,7 +37,12 @@ func (c *prometheusClientMock) setResponce(key types.NamespacedName, stats *Volu
 
 var _ = Describe("test prometheusClient", func() {
 	It("test metrics", func() {
-		c, err := NewPrometheusClient("http://noconnection")
+		ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			http.NotFound(w, r)
+		}))
```

Comment:
It can be more simplified as `httptest.NewServer(http.HandlerFunc(http.NotFound))`

Reply:
fixed it.